src/prelude.rs

```diff
@@ -1,12 +1,5 @@
+#[doc(no_inline)]
 pub use crate::{
-    document::DocumentOps,
-    group::GroupOps,
-    internal::{
-        document_api::{DocumentId, DocumentName},
-        group_api::{GroupId, GroupName},
-        user_api::{DeviceId, DeviceName, UserId},
-    },
-    policy::PolicyGrant,
-    user::UserOps,
-    DeviceContext, IronOxide, IronOxideErr,
+    common::*, config::*, document::advanced::*, document::*, group::*, policy::*, user::*,
+    InitAndRotationCheck, IronOxide, IronOxideErr, PrivateKeyRotationCheckResult,
```

Comment:
I can't wildcard re-export the top-level exports because that would force our custom `Result` type on anyone who imported the prelude.